docs/USAGE-GUIDE.md

```diff
@@ -830,63 +830,23 @@ A PSK must not be shared between more than one server and one client. An entity
 
 ### Configuring External Pre-Shared Keys
 
-Use the following APIs to configure external pre-shared keys.
+Both clients and servers will need to create and append a PSK to a connection in order to negotiate that PSK. Call `s2n_external_psk_new()` to allocate memory for a PSK object. Call `s2n_psk_set_identity()` to set a unique identifier for the PSK. Note that this identity will be transmitted over the network unencrypted, so do not include any confidential information in it. Call `s2n_psk_set_secret()` to set the secret value for a given PSK. Deriving a shared secret from a password or other low-entropy source is _not_ secure and is subject to dictionary attacks. Call `s2n_psk_set_hmac()` to change the default PSK hmac algorithm (**S2N_PSK_HMAC_SHA256**) for a given PSK. Note that the hmac algorithm may influence server cipher-suite selection.
```

Comment:
Can you write "pre-shared key (PSK)" the first time you use the term?

"to change the default PSK hmac algorithm (**S2N_PSK_HMAC_SHA256**) for a given PSK" -> "to change the hmac algorithm (defaults to **S2N_PSK_HMAC_SHA256**) for a given PSK"